pkg/ruler/rulespb/compat_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/grafana/mimir/pkg/mimirpb"
```

Comment:
`github.com/grafana/mimir` imports need to be in their own group beneath 3rd part imports

Reply:
yeah i noticed indeed, lemme update accordingly